### PR TITLE
Fix glxgears poo#15578

### PR DESCRIPTION
**Pull request by okurz**
* Bump up network involving initialization steps
* Simplify ImageMagick redundancy
* Use pkcon consistently in x11 tests
* Ensure packagekit is available on use of ensure_installed

Local verification run: http://lord.arch/tests/5509

### CODE REVIEW

**Comment by okurz** [comment]
waiting for verification run http://lord.arch/tests/5509 to finish

**Comment by okurz** [comment]
done, passed. please review and merge

File: lib/susedistribution.pm [hunk 1]
```diff
@@ -136,6 +136,8 @@ sub ensure_installed {
     assert_screen('xterm');
     testapi::assert_script_sudo("chown $testapi::username /dev/$testapi::serialdev");
     my $retries = 5;    # arbitrary
+                        # make sure packagekit service is available
```
Comment (coolo) [inline lib/susedistribution.pm]: that's quite strange comment placement

File: tests/x11/ghostscript.pm [hunk 2]
```diff
@@ -21,20 +21,9 @@ use testapi;
...
-    pkcon_quit;
-    zypper_call "in ghostscript ghostscript-x11";
-
-    # special case for gv which is not installed on all flavors
-    my $gv_missing = zypper_call("in gv", exitcode => [0, 104]);
-
-    # exit root shell
-    type_string "exit\n";
+    select_console 'x11';
+    ensure_installed 'ghostscript ghostscript-x11 gv';
```
Comment (coolo) [inline tests/x11/ghostscript.pm]: you can't ensure_installed what does not exist </iron_men>

**Comment by coolo** [comment]
CC @rwmanos

File: tests/x11/ghostscript.pm (hunk 2 above)
Comment (okurz) [inline tests/x11/ghostscript.pm]: @rwmanos can you give me an example product where `gv` would not be available so that I could experience that myself?

Comment (rwmanos) [inline tests/x11/ghostscript.pm]: @okurz I don't know any. Maybe we can ask @eisfuchs-de who is the maintainer.

**Comment by rwmanos** [comment]
Thanks for the CC. The changes to the ImageMagick test seem really good 👍

File: lib/susedistribution.pm (hunk 1 above)
Comment (okurz) [inline lib/susedistribution.pm]: fixed

File: tests/x11/ghostscript.pm (hunk 2 above)
Comment (okurz) [inline tests/x11/ghostscript.pm]: should we just merge and see where it fails then?

Comment (eisfuchs-de) [inline tests/x11/ghostscript.pm]: I believe that we ran into issues on sled where gv was not installed but it was available.

Comment (coolo) [inline tests/x11/ghostscript.pm]: sled has no gv

Comment (coolo) [inline tests/x11/ghostscript.pm]: and then if it's available in SLED13 we still don't test it. Why not leave the test smart?

**Comment by okurz** [comment]
ok, so I have removed the commit to use `ensure_installed` for now. As you seem to be fine with all other parts I will merge this now and keep the other commit for later and make the module more "smart" again

**Pull request merged**